api/Makefile

```diff
@@ -18,8 +19,8 @@ stop:
 # Run tests for the library
 test:
 	docker compose up -d --build
-	docker cp requirements-dev.txt api_web_1:/app/requirements-dev.txt
+	docker cp requirements-dev.txt api_web:/app/requirements-dev.txt
```

Comment:
So, it was completely broken here ? Container name is defined `api_web` in `docker-compose.yml` so the command was not working before ?

Reply:
Mh in the CI it is already crashed with the docker-compose command and an unexpected path .. unfortunately this has still lead to a green job :sweat_smile: 

Now it works locally and within the CI